settings.d/pulp3.yml.example

```diff
@@ -0,0 +1,9 @@
+---
+:enabled: true
+#:pulp_url: https://localhost/pulp/
+# Path to pulp, pulp content and mongodb directories
```

Comment:
This comment seems redundant.

Reply:
removed, and also realized we didn't need mongo settings for pulp3 :)